Base/res/html/misc/new-tab.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>New Tab</title>
+    <style>
+        #container {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+
+        #main {
+            text-align: center;
+            display: block;
+            width: 100%;
+            max-width: 400px;
+        }
+
+        #image {
+            image-rendering: pixelated;
+        }
+
+        #input {
+            width: 100%;
+            padding: 5px;
+        }
+
+        #search-buttons {
+            display: flex;
+            justify-content: space-between;
+        }
+
+        button:hover {
+            cursor: pointer;
+        }
+    </style>
+</head>
+<body>
+    <div id="container">
+        <div id="main">
+            <br>
+            <img src="/res/icons/32x32/app-browser.png" width="64" height="64" id="image"><br><br>
+            <form>
+                <input type="search" name="q" id="input"><br><br>
```

Comment:
Likewise - style/access `input[type=search]` instead, or be more specific than `input` with the id name